src/ode_solver/explicit_ode_solver.cpp

```diff
@@ -68,6 +72,18 @@ void RungeKuttaODESolver<dim,real,n_rk_stages,MeshType>::step_in_time (real dt,
             
         this->dg->solution = this->rk_stage[i];
 
+        // Apply limiter at every RK stage
+        if (limiter_type != limiter_enum::none || apply_tvb == true) {
```

Comment:
this logic might be more appropriate to be checked in the `create_limiter` factory; reducing this to simply `if(apply_tvb)`